test/compflags/ferguson/unstable-class.compopts

```diff
@@ -1 +1,2 @@
---warn-unstable
+--warn-unstable # unstable-class.good
```

Comment:
Will updating adding --no-force-initializers to this line avoid a failure with --force-initializers?